src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -77,13 +77,19 @@ class NexusPublishPlugin : Plugin<Project> {
             val closeAndReleaseTask = project.tasks.register<Task>(
                     "closeAndRelease${capitalizedName}StagingRepository")
             closeTask {
+                setDescription("Closes open staging repository in '$name' Nexus instance.")
```

Comment:
I couldn't use `descrption = ".."` here, as the [staging repository descrption](https://github.com/gradle-nexus/publish-plugin/blob/002d3d55eb66c71e3736a44a48673222e3791e25/src/main/kotlin/io/github/gradlenexus/publishplugin/AbstractNexusStagingRepositoryTask.kt#L49) overrides the one from `AbstractTask`. I wonder if it could be a problem in the future?

Reply:
Good catch, I forgot about `Task.description`. 🤦 I'll rename our property to `repositoryDescription`.

Reply:
```suggestion
                description = "Closes open staging repository in '$name' Nexus instance."
```